src/proxy.c

```diff
@@ -1383,10 +1384,7 @@ cc_proxy_run_hyper_new_container (struct cc_oci_config *config,
 	json_object_set_string_member (process, "workdir",
 			config->oci.process.cwd);
 
-	/* Hyperstart is not able to handle uid 0. This is a bug.
-	 * Simply dont sent the uid for now as the container
-	 * runs as root by default until the bug is resolved.
-	 */
+	// Hyperstart is not able to handle uid 0. This is a bug.
 	if ((int)config->oci.process.user.uid != 0) {
```

Comment:
Note strictly part of this PR, but it would be *much* clearer to my mind if the logic was:

```
if ((int)config->oci.process.user.uid -= 0) {
    json_object_set_string_member (process, "user", "root");
} else {
   ...
}

```